script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -364,14 +363,12 @@ This bedrock was the true objective.
 [sha:6c00d220196cb605f57f47829cd01f4a8ac1929b]{
 -- Page 680, Offset 10490.
 -- 吸血鬼の視界から外れ、<超|・><低|・><温|・><の|・><影|・><響|・><を|・><受|・><け|・><な|・><い|・><ヤ|・><ツ|・><の|・><頭|・><上|・><に|・>、俺を縛り付けてふっ飛ばしたのだ。
-// 吸血鬼の視界から外れ、<超|・><低|・><温|・><の|・><影|・><響|・><を|・><受|・><け|・><な|・><い|・><ヤ|・><ツ|・><の|・><頭|・><上|・><に|・>、俺を縛り付けてふっ飛ばしたのだ。
-She tied me down and blasted me out of his field of view, to a place unaffected by his cryogenic influence.
+She tied me down and blasted me out of his field of view, %{i}to a place unaffected by his cryogenic influence%{/i}.
```

Comment:
Massive blocks of italics like this just don't feel natural

Reply:
I think these italics are unnecessary anyways